Userland/Libraries/LibCore/ArgsParser.cpp

```diff
@@ -595,6 +595,49 @@ void ArgsParser::add_option(Vector<size_t>& values, char const* help_string, cha
     add_option(move(option));
 }
 
+void ArgsParser::add_option(Optional<ssize_t>& value, char const* help_string, char const* long_name, char short_name, char const* value_name, OptionHideMode hide_mode)
+{
+    Option option {
+        OptionArgumentMode::Required,
+        help_string,
+        long_name,
+        short_name,
+        value_name,
+        [&value](StringView s) -> ErrorOr<bool> {
+            value = s.to_number<ssize_t>();
+            return value.has_value();
+        },
+        hide_mode,
+    };
+    add_option(move(option));
+}
+
+void ArgsParser::add_option(Vector<ssize_t>& values, char const* help_string, char const* long_name, char short_name, char const* value_name, char separator, OptionHideMode hide_mode)
+{
+    Option option {
+        OptionArgumentMode::Required,
+        help_string,
+        long_name,
+        short_name,
+        value_name,
+        [&values, separator](StringView s) -> ErrorOr<bool> {
+            bool parsed_all_values = true;
+
+            s.for_each_split_view(separator, SplitBehavior::Nothing, [&](auto value) {
+                if (auto maybe_value = AK::StringUtils::convert_to_uint<ssize_t>(value); maybe_value.has_value())
```

Comment:
As noted by the CI failure, you can't use `convert_to_uint` for signed ints. `value.to_number<ssize_t>()` should be fine I think.

Reply:
I used convert_to_int now